### PR TITLE
Initial version of CDT.cloud Blueprint

### PR DESCRIPTION
**Pull request by sdirix**
#### What it does

Initial version of the CDT.cloud Blueprint using the up-to-date Theia Blueprint as a basis.

#### How to test
`yarn && yarn electron start`

#### State

- Logos are not yet replaced
- Only tested on Linux, Mac and Windows not tested yet
- Not yet integrated into Jenkins (although preparations are done)
- VSCode extensions need to be updated (Open VSX is down at the moment)

Plan is to squash-merge the contribution for a clean commit

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,62 +0,0 @@
-<div id="theia-logo" align="left">
-    <br />
-    <img src="https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/EF_GRY-OR_svg.svg?sanitize=true" alt="Eclipse Logo" width="300"/>
-</div>
-
-# Community Code of Conduct
```
Comment (sdirix) [inline CODE_OF_CONDUCT.md:6]: @planger I deleted this file as we don't have a CoC for CDT.cloud yet.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,123 +0,0 @@
-# Contributing to Eclipse Theia
```
Comment (sdirix) [inline CONTRIBUTING.md:1]: @planger We don't have something similar yet for CDT.cloud so I deleted this.

File: applications/electron/package.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-  "productName": "Theia Blueprint",
+  "name": "cdt-cloud-blueprint",
+  "description": "CDT.cloud Blueprint is a template for building custom web-based C/C++ tools",
+  "productName": "CDT.cloud Blueprint",
   "version": "1.22.1",
   "main": "scripts/theia-electron-main.js",
   "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
-  "author": "Eclipse Theia <theia-dev@eclipse.org>",
-  "homepage": "https://github.com/eclipse-theia/theia-blueprint#readme",
+  "author": "CDT.cloud <sdirix@eclipsesource.com>",
```
Comment (sdirix) [inline applications/electron/package.json:9]: @planger Is there some generic mail for cdt cloud which somebody checks we can use? Or shall we keep it my mail for now?

Comment (planger) [reply applications/electron/package.json:9]: Do we have to provide an e-mail address in `author`? In GLSP we use just `author.name`. We use the dev mailing list of the Eclipse project in `contributors` though.

```json
"author": {
  "name": "Eclipse GLSP"
},
"contributors": [
  {
    "name": "Eclipse GLSP Project",
    "email": "glsp-dev@eclipse.org",
    "url": "https://projects.eclipse.org/projects/ecd.glsp"
  }
]
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (planger) [reply CONTRIBUTING.md:1]: Right, we should add a dedicated `CONTRIBUTING.md` later.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (planger) [reply CODE_OF_CONDUCT.md:6]: Imho we should add the CoC of Eclipse to all CDT.cloud repositories.
https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php
But this doesn't have to be done right now with this initial contribution.

**Comment by eneufeld** [comment]
changes look good to me, the open questions you raised must be addressed of course

File: applications/electron/package.json (hunk 3 above)
Comment (sdirix) [reply applications/electron/package.json]: done

**Review by planger — APPROVED** [review]
Looks great, thanks a lot! :+1:

**Pull request merged**